docs/reference/manifest/control.md

```diff
@@ -31,9 +31,12 @@ A button performs a single action when the user selects it. It can either execut
 |  [Supertip](supertip.md)  | Yes |  The supertip for the button.    |
 |  [Icon](icon.md)      | Yes |  An image for the button.         |
 |  [Action](action.md)    | Yes |  Specifies the action to perform.  |
+|  [Enabled](enabled.md)    | No |  Specifies whether the control is enabled when the add-in launches.  |
 
 ### ExecuteFunction button example
 
+In the following example, the button is disabled when the add-in launches. It can be programmatically enabled. For more information, see [Enable and Disable Add-in Commands](/office/dev/add-ins/design/disable-add-in-commands).
```

Comment:
Nit: Since this is in the same repo, it should be a relative link (../design/disable-add-in-commands.md)